DataGenerator/src/main/java/org/opensearch/migrations/data/WorkloadOptions.java

```diff
@@ -0,0 +1,21 @@
+package org.opensearch.migrations.data;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.opensearch.migrations.data.workloads.Workloads;
+
+import com.beust.jcommander.Parameter;
+
+public class WorkloadOptions {
+    @Parameter(names = { "--workloads", "-w" }, description = "The list of workloads to run, defaults to all available workloads.", required = false)
```

Comment:
Mind adding the names of the possible workloads either here or in the README, or both?  Just thinking a user wouldn't really know what the options were.

Reply:
Thanks, I'll update the readme.

Props to jcommander, it is included in the help output
>   --workloads, -w
      The list of workloads to run, defaults to all available workloads.
      Default: [Geonames, HttpLogs, Nested, NycTaxis]